lib/supa/command.rb

```diff
@@ -16,8 +16,12 @@ def represent
     private
 
     def value
-      return @representer.send(@options[:modifier], apply_render_flags(raw_value)) if @options[:modifier]
-      apply_render_flags(raw_value)
+      return @value if defined?(@value)
+      @value = modifier ? @representer.send(modifier, apply_render_flags(raw_value)) : apply_render_flags(raw_value)
```

Comment:
@Strech Applied 1 and 3 https://github.com/distribusion/supa/pull/20/commits/693c0110e07c215e181b7d796000e3969674482b. I didn't introduce Proc support because I fell that we try to do to many things here and it is not the main thing that was supposed to be done here (hide_when_nil).